src/wrap.c

```diff
@@ -0,0 +1,44 @@
+#include <curses.h>
+
+#ifndef NCURSES_VERSION
+/*
+ * Some things are declared differently in NetBSD Curses; these are handled here
+ */
+
+int (COLOR_PAIR)(int n) {
+	return COLOR_PAIR(n);
+}
```

Comment:
How doesn't this infinitely recurse?

Reply:
Quoting c99 spec 7.1.4.5 referring to how to guarantee the usage of a true function implementation rather than a macro implementation in the standard library.
> byuse of its associated header (assuredly generating a true function reference)
> ```c
> #include <stdlib.h>
> #undef atoiconst char *str;
> /*...*/
> i=atoi(str);
> ```
> or
> ```c
> #include <stdlib.h>
> const char *str;
> /*...*/
> i=(atoi)(str);
> ```